tests/unit/aggregate-step-form.spec.ts

```diff
@@ -24,17 +24,61 @@ describe('Aggregate Step Form', () => {
       expect(widgetMultiselect.attributes('options')).toEqual('columnA,columnB,columnC');
     });
 
+    it('keepOriginalGranularity should be set properly if defined in inititalStepValue', async () => {
+      const wrapper = runner.shallowMount(undefined, {
+        propsData: {
+          initialStepValue: {
+            name: 'aggregate',
+            on: [''],
+            aggregations: [],
+            keepOriginalGranularity: true,
+          },
+        },
+      });
+      await wrapper.vm.$nextTick();
+      expect(wrapper.vm.$data.editedStep.keepOriginalGranularity).toEqual(true);
+    });
+
+    it('keepOriginalGranularity should be set to false if undefined in inititalValue', async () => {
+      const wrapper = runner.shallowMount(undefined, {
+        propsData: {
+          initialStepValue: {
+            name: 'aggregate',
+            on: [''],
+            aggregations: [],
+            keepOriginalGranularity: false,
```

Comment:
```suggestion
            keepOriginalGranularity: undefined,
```
From there, we don't need any `keepOriginalGranularity: false,` in this file

Reply:
👍

Reply:
I only kept it in 1 test afterwards, (`runner.testValidate()`) that checks that we emit the same step as the input